bazel/foreign_cc/luajit.patch

```diff
@@ -86,7 +86,7 @@ index 0000000..9c71271
 +      os.environ["HOST_CFLAGS"] = "-fno-sanitize=memory"
 +      os.environ["HOST_LDFLAGS"] = "-fno-sanitize=memory"
 +
-+    # Blacklist LuaJIT from ASAN for now.
++    # Remove LuaJIT from ASAN for now.
 +    # TODO(htuch): Remove this when https://github.com/envoyproxy/envoy/issues/6084 is resolved.
 +    if "ENVOY_CONFIG_ASAN" in os.environ:
 +      os.environ["TARGET_CFLAGS"] += " -fsanitize-blacklist=%s/com_github_luajit_luajit/clang-asan-blacklist.txt" % os.environ["PWD"]
```

Comment:
change the file name too?

Reply:
I figured since the compiler flag has blacklist in it we should keep it but I can change it. Will do.